### PR TITLE
Make possible to overwrite readme file location and type

### PR DESCRIPTION
**Pull request by kokosing**
Make possible to overwrite readme file location and type

### CODE REVIEW

File: airbase/pom.xml [hunk 1]
```diff
@@ -197,6 +197,10 @@
 
         <!-- Checkstyle -->
         <air.checkstyle.config-file>checkstyle/airbase-checks.xml</air.checkstyle.config-file>
+
+        <!-- Readme -->
```
Comment (electrum) [inline airbase/pom.xml]: We should be more specific:
```suggestion
        <!-- Readme for launcher -->
```

**Comment by electrum** [comment]
What's the motivation for this change? I think there was a specification for the tarball format.

cc @martint

**Comment by kokosing** [comment]
> What's the motivation for this change?

We use airbase in one of projects and we had dummy README.txt (and proper README.md) just to supply this check this. Not a big deal, but it would be nice to attach proper readme file.

File: airbase/pom.xml [hunk 2]
```diff
@@ -198,7 +198,7 @@
         <!-- Checkstyle -->
         <air.checkstyle.config-file>checkstyle/airbase-checks.xml</air.checkstyle.config-file>
 
-        <!-- Readme -->
+        <!-- Readme for launcher -->
```
Comment (kokosing) [inline airbase/pom.xml]: This is going to be squashed.

**Comment by kokosing** [comment]
Updated

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**